.devcontainer/.env

```
@@ -0,0 +1,67 @@
+ZMS_TICKETPRINTER_TWIG_CACHE=false
+
+# Cache Configuration
+CACHE_DIR=/var/www/html/zmscitizenapi/cache
+SOURCE_CACHE_TTL=3600
+
+# Maintenance Mode
+MAINTENANCE_ENABLED=false
+
+# Logger Configuration
+LOGGER_MAX_REQUESTS=1000
+LOGGER_RESPONSE_LENGTH=1048576  # 1MB
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)